Cargo.lock

```diff
@@ -6855,9 +6855,18 @@ dependencies = [
 
 [[package]]
 name = "tinyvec"
-version = "0.3.4"
```

Comment:
found this in crates.io: https://crates.io/crates/tinyvec/versions
but did not find this version in the repository: https://github.com/Lokathor/tinyvec/tags?after=v0.4.0

Reply:
the diff between the two packages (0.3.3 and 0.3.4) looks unsubstantial, but still a bit strange:
```
Only in /tmp/tmp8x8pxdjw/output_dir_0.3.3/tinyvec-0.3.3: .cargo_vcs_info.json
Only in /tmp/tmp8x8pxdjw/output_dir_0.3.3/tinyvec-0.3.3: .gitignore
Only in /tmp/tmp8x8pxdjw/output_dir_0.3.3/tinyvec-0.3.3: .travis.yml
diff -r /tmp/tmp8x8pxdjw/output_dir_0.3.3/tinyvec-0.3.3/Cargo.toml /tmp/tmp8x8pxdjw/output_dir_0.3.4/tinyvec-0.3.4/Cargo.toml
16c16
< version = "0.3.3"
---
> version = "0.3.4"
21c21
< license = "Zlib"
---
> license = "Zlib OR Apache-2.0 OR MIT"
```